examples/cluster/kubeproxy/kubeproxy.yaml

```diff
@@ -43,7 +43,7 @@ spec:
         - kube-proxy {{cluster_cidr}} --resource-container="" --oom-score-adj=-998 {{params}} 1>>/var/log/kube-proxy.log 2>&1
         env:
         - name: KUBERNETES_SERVICE_HOST
-          value: "{{kubernetes_service_host_env_value}}"
+          value: "\"{{kubernetes_service_host_env_value}}\""
```

Comment:
If the existing quotes get removed during template processing, does this result in `value: \"10.10.10.10\"` in the rendered template, or `value: "10.10.10.10"`?

Reply:
On the master node I found something like:

```
- name: KUBERNETES_SERVICE_HOST
- value: fd20:2001:1::
```

Though apparently it's not because of this file, there is rather a different copy that needs a fix instead.

Reply:
Thanks for your offline investigation @mtaufen!